### ISSUE TITLE
what is the minimum block rate?

### ISSUE THREAD

**Issue by sack2116** [mdesilva/AmazonFlexUnlimited#62]
what is the minimum block rate? and how does it defer from  "minPayRatePerHour": ?

thanks!

**Comment by gmcsnetwork** [comment]
Hello @sack2116,
Were you able to find more information on what setting "minBlockRate" is? I'm looking for the same information. 

Thank you

**Comment by gmcsnetwork** [comment]
Actually, I was just reading through the pull request and found the following information on "minBlockRate".

_"Adding an option to only accept offers that are greater than or equal to a certain amount of pay, minBlockRate"_

**Comment by sack2116** [comment]
> @sack2116 ,
> 
> Actually, I was just reading through the pull request and found the following information on "minBlockRate".
> 
> _"Adding an option to only accept offers that are greater than or equal to a certain amount of pay, minBlockRate"_

Yeah, I read the same thing... but when you combine this with the payrateperHour, having those 2 set I don't know how I can go about just picking offers base on a particular price or price range,  like for example if offers base pay is = or less than. Then accept it.

**Comment by sack2116** [comment]
Having a minpayperhour is messing up how I want the bot to operate, because all blocks have different times for delivery , so it is hard to find a min lay per hour that takes intl consideration all different block timeframes... like 1 hr blocks vs 4 hr blocks.. etc

**Comment by russbakr** [comment]
looks like (in order of importance) - 

minBlockRate first (then)
minPayRatePerHour second

say you have a minBlockRate of $100 it will attempt that no matter block hours
and then your pay per hour $25 per hour / blockrate hour

**Comment by sack2116** [comment]
how can make it default to just the whole foods market, which normally are of lower block rate and would often fall outside of 19/hour ?
